hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-router/src/main/java/org/apache/hadoop/yarn/server/router/clientrm/RouterYarnClientUtils.java

```diff
@@ -126,50 +126,55 @@ private static ApplicationReport mergeUAMWithUAM(ApplicationReport uam1,
   }
 
   private static void mergeAMWithUAM(ApplicationReport am,
-      ApplicationReport uam){
+                                     ApplicationReport uam){
     ApplicationResourceUsageReport amResourceReport =
-        am.getApplicationResourceUsageReport();
+            am.getApplicationResourceUsageReport();
```

Comment:
Can you avoid all this whitespace changes?
This one in particular goes against the Hadoop style.

Reply:
I have applied the Hadoop style to fix the whitespace issues. However, since the line mapping has changed and due to the indentation changes because of new `else if`, it would show some whitespace changes. Please let me know if anything else needs to be fixed here.